app/jobs/send_invitations_job.rb

```diff
@@ -103,12 +103,20 @@ def filled_out_mentor_voormeting_last_week?(response)
   end
   # rubocop:enable Metrics/AbcSize
 
+  def one_time_reminder_window
+    Time.zone.now > Time.new(2018, 5, 17, 9).in_time_zone &&
+      Time.zone.now < Time.new(2018, 5, 21).in_time_zone
+  end
+
   # rubocop:disable Metrics/CyclomaticComplexity
   # rubocop:disable Metrics/PerceivedComplexity
   # rubocop:disable Metrics/MethodLength
   # rubocop:disable Metrics/AbcSize
   def mentor_texts(response)
-    if filled_out_mentor_voormeting_last_week?(response)
+    if one_time_reminder_window
```

Comment:
het is denk ik by design dat dit niet exact dezelfde tekst is als de 'normale' invite sms? dit: `"Hoi #{target_first_name(response)}, je wekelijkse vragenlijsten staan weer voor je klaar!"`.